profiler/options.go

```diff
@@ -214,10 +215,12 @@ func defaultConfig() (*config, error) {
 	}
 	// If DD_PROFILING_ENABLED is set to "auto", the profiler's activation will be determined by
 	// the Datadog admission controller, so we set it to true.
-	if os.Getenv("DD_PROFILING_ENABLED") == "auto" {
+	// TODO: APMAPI-1358
+	if v, _ := stableconfig.StringStableConfig("DD_PROFILING_ENABLED", ""); v == "auto" {
 		c.enabled = true
 	} else {
-		c.enabled = internal.BoolEnv("DD_PROFILING_ENABLED", true)
+		// TODO: APMAPI-1358
```

Comment:
Looks like this ticket is about a system test. What would we need for that?

Reply:
Ah apologies, this was probably unclear.
The "TODO" is to report `Origin` to telemetry, which is part of the next phase of work. I referenced the jira card APMAPI-1358 because:
- it references the test that checks against `Origin` reporting to telemetry, so is shorthand for "todo: implement behavior to pass this test"
- Using a singular key like APMAPI-1358 is a good way for me to go back to the code base and command +f for all the spots where I need to report Origin

Let me know if this is too confusing and I can change it, or remove it entirely for now

Reply:
Adding this to the PR description as well.

Reply:
Makes sense to me, thanks for the explanation!